code/__HELPERS/text.dm

```diff
@@ -92,8 +92,12 @@
 	if(non_whitespace)		return text		//only accepts the text if it has some non-spaces
 
 // Used to get a sanitized input.
-/proc/stripped_input(var/mob/user, var/message = "", var/title = "", var/default = "", var/max_length=MAX_MESSAGE_LEN)
-	var/name = input(user, message, title, default)
+/proc/stripped_input(var/mob/user, var/message = "", var/title = "", var/default = "", var/max_length = MAX_MESSAGE_LEN, type = "input")
+	var/name
+	if(type == "input")

```

Comment:
I think these changes would probably be better as a seperate proc, given how short this proc previously was.